demisto_sdk/__main__.py

```diff
@@ -2892,13 +2896,18 @@ def error_code(config, **kwargs):
 )
 @click.help_option("-h", "--help")
 @click.option(
-    "-ud",
-    "--use-docker",
-    is_flag=True,
-    help="Use docker service to run the content graph",
+    "-o",
+    "--output-path",
+    type=click.Path(resolve_path=True, path_type=Path, dir_okay=True, file_okay=False),
+    default=None,
+    help="Output folder to place the zip file of the graph exported CSVs files",
 )
 @click.option(
-    "-us", "--use-existing", is_flag=True, help="Use existing service", default=False
+    "-mp",
+    "--marketplace",
+    help="The marketplace to generate the graph for.",
+    default="xsoar",
+    type=click.Choice([marketplace.value for marketplace in MarketplaceVersions]),
```

Comment:
```suggestion
    type=click.Choice(list(MarketplaceVersions)),
```
pretty sure it works, especially with `str, Enum`